### DISCUSSION TITLE
Does v1.26.0 *really* need Grafana 8.3.4? Version 8.3.3 is not enough?

### DISCUSSION THREAD

**Discussion by jmalcaide** [teslamate-org/teslamate#2374] [Q&A] [answered]
I am running a manual install of Teslamate on TrueNAS 12, inside a jail instance based on FreeBSD 12.2. I can easily install Grafana 8.3.3 using FreeBSD packages, while for installing version 8.3.4 I need to build the port (with a very slow CPU).

**Comment by adriankumpf** [comment, marked as answer]
Let's say it's recommended for the best experience but not required per se.

We need this fix which shipped with [v8.3.4](https://github.com/grafana/grafana/releases/tag/v8.3.4): https://github.com/grafana/grafana/pull/43376 Otherwise the tables will be messed up.

**Reply by jmalcaide** [reply]
OK, I will try with Grafana 8.3.3 and see what happens. Thank you!
